src/librustc_mir/borrow_check/type_check/mod.rs

```diff
@@ -686,6 +686,7 @@ impl<'a, 'b, 'tcx> TypeVerifier<'a, 'b, 'tcx> {
                 let fty = self.sanitize_type(place, fty);
                 match self.field_ty(place, base, field, location) {
```

Comment:
Is `field_ty` used in more places? Do those places normalize too? Maybe it would be good to normalize inside `field_ty` itself.

Reply:
The other call to `field_ty` happens here https://github.com/spastorino/rust/blob/7bafb5704d9727b3897f086d9f159e62bc40ce42/src/librustc_mir/borrow_check/type_check/mod.rs#L1182-L1183. I can move the `normalize` call inside it but are just those two.

Reply:
btw, I can move both `normalize` calls inside `field_ty` if you prefer.